web_src/js/features/repo-issue-list.js

```diff
@@ -119,8 +120,69 @@ function initRepoIssueListAuthorDropdown() {
   };
 }
 
+function initPinRemoveButton() {
+  for (const button of document.getElementsByName('issue-unpin-button')) {
+    button.addEventListener('click', (event) => {
+      const el = event.currentTarget;
+      const id = Number(el.getAttribute('data-issue-id'));
+
+      // Send the unpin request
+      fetch(el.getAttribute('data-unpin-url'), {
+        method: 'delete',
+        headers: {
+          'X-Csrf-Token': window.config.csrfToken,
+          'Content-Type': 'application/json',
+        },
+      }).then((response) => {
+        if (response.ok) {
+          // Delete the tooltip
+          el._tippy.destroy();
+          // Remove the Card
+          document.querySelector(`div[data-issue-id="${id}"]`).remove();
+        }
+      });
+    });
+  }
+}
+
+function pinMoveEnd(e) {
+  const url = e.item.getAttribute('data-move-url');
+  const id = Number(e.item.getAttribute('data-issue-id'));
+  fetch(url, {
+    method: 'post',
+    body: JSON.stringify({id, position: e.newIndex + 1}),
+    headers: {
+      'X-Csrf-Token': window.config.csrfToken,
+      'Content-Type': 'application/json',
+    },
+  });
+  initPinRemoveButton();
```

Comment:
duplicate call to initPinRemoveButton ? some element may have duplicate `click` events?

Reply:
I initially though the Icon would lose the Event when the div is moved, but is appears this is not true, so I removed it now